pkg/api/handlers/libpod/images_pull.go

```diff
@@ -36,6 +38,7 @@ func ImagesPull(w http.ResponseWriter, r *http.Request) {
 		AllTags    bool   `schema:"allTags"`
 		PullPolicy string `schema:"policy"`
 		Quiet      bool   `schema:"quiet"`
+		Progress   bool   `schema:"progress"`
```

Comment:
The new parameter needs wiring in `pkg/api/server` for the Swagger docs and wiring for `podman-remote` (and `pkg/bindings`).

Reply:
Wait, wait, it's old code :)

Reply:
I will have a new param though, so these updates will be needed.